presets/crt-guest-advanced-ntsc.slangp

```diff
@@ -0,0 +1,134 @@
+shaders = 17
+
+shader0 = ../stock.slang
+filter_linear0 = false
+scale_type0 = source
+scale0 = 1.0
+
+shader1 = ../stock.slang
+filter_linear1 = false
+scale_type1 = source
+scale1 = 1.0
+alias1 = StockPass
+
+shader2 = ../crt/shaders/guest/advanced/afterglow0.slang
+filter_linear2 = true
+scale_type2 = source
+scale2 = 1.0
+alias2 = AfterglowPass
+
+shader3 = ../crt/shaders/guest/advanced/pre-shaders-afterglow.slang
+filter_linear3 = true
+scale_type3 = source
+scale3 = 1.0
+alias3 = PrePass0 
+
+textures = "SamplerLUT1;SamplerLUT2;SamplerLUT3;SamplerLUT4"
+SamplerLUT1 = ../crt/shaders/guest/advanced/lut/trinitron-lut.png
+SamplerLUT1_linear = true
+SamplerLUT2 = ../crt/shaders/guest/advanced/lut/inv-trinitron-lut.png
+SamplerLUT2_linear = true
+SamplerLUT3 = ../crt/shaders/guest/advanced/lut/nec-lut.png
+SamplerLUT3_linear = true
+SamplerLUT4 = ../crt/shaders/guest/advanced/lut/ntsc-lut.png
+SamplerLUT4_linear = true 
+
+# custom ntsc shaders
+
+shader4 = ../ntsc/shaders/ntsc-adaptive/ntsc-pass1.slang
+shader5 = ../ntsc/shaders/ntsc-adaptive/ntsc-pass2.slang
+
+filter_linear4 = false
+filter_linear5 = true
+
+scale_type_x4 = source
+scale_type_y4 = source
+scale_x4 = 4.0
+scale_y4 = 1.0
+frame_count_mod4 = 2
```

Comment:
Is this frame_count_mod line necessary to add into every presets using ntsc-adapt now?

Reply:
No, it's already mod()ed in the shader, so it's extraneous.